app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/rules/ExceptionRulesDebugReceiver.kt

```diff
@@ -86,22 +86,19 @@ class ExceptionRulesDebugReceiverRegister @Inject constructor(
     private val shouldSaveRules = AtomicBoolean(true)
 
     override fun onVpnStarted(coroutineScope: CoroutineScope) {
-        // only for debug builds
-        if (appBuildConfig.isDebug) {
```

Comment:
Removed condition as this functionality should be the same in debug and release. This is only used in internal builds. The rest of the code in this function is untouched.